Max-Movies-In-Years.js

```diff
@@ -0,0 +1,40 @@
+const res = [
```

Comment:
Let's remove this file. Always try to get rid of stray code that's not serving purpose